### PR TITLE
chore: update polyjuice dep refs

### LINKED ISSUE #875: GWOS EVM MainCI Error
image
https://github.com/godwokenrises/godwoken/actions/runs/3553903432/workflow#L66

### PR DESCRIPTION
**Pull request by magicalne**
Make polyjuice reference deps in the monorepo.

### CODE REVIEW

File: gwos-evm/README.md [hunk 1]
```diff
@@ -100,3 +100,22 @@ Polyjuice only provides contract accounts. Godwoken's user account is leveraged
...
 * [Life of a Polyjuice Transaction](https://github.com/nervosnetwork/godwoken/blob/develop/docs/life_of_a_polyjuice_transaction.md)
 * [Life of a Godwoken Transaction](https://github.com/nervosnetwork/godwoken/blob/develop/docs/life_of_a_godwoken_transaction.md)
+
+## Development
+
+### Build
+
+```sh
+cargo install moleculec --version 0.7.2
+git submodule update --init  --recursive
```
Comment (Flouse) [inline gwos-evm/README.md]:
```suggestion
git submodule update --init --recursive --depth=1
```

File: gwos-evm/Makefile [hunk 2]
```diff
@@ -9,6 +9,7 @@ OBJCOPY := $(TARGET)-objcopy
 
 SECP_DIR := deps/secp256k1-fix
 SECP256K1_SRC := $(SECP_DIR)/src/ecmult_static_pre_context.h
+GW_SCRIPTS := deps/gwos/c
```
Comment (Flouse) [inline gwos-evm/Makefile]:
```bash
@Flouse ➜ /workspaces/godwoken/gwos-evm (poly_ref ✗) $ make dist

# error
cat: deps/gwos/c/Makefile: No such file or directory
cat: deps/gwos/c/Makefile: No such file or directory
```

**Review by Flouse — CHANGES_REQUESTED** [review]

**Review by keroro520 — DISMISSED** [review]

**Comment by magicalne** [comment]
CI was fixed.

File: .github/workflows/gwos-evm-ci.yml [hunk 3]
```diff
@@ -66,12 +66,11 @@ jobs:
     uses: nervosnetwork/godwoken-tests/.github/workflows/reusable-integration-test-v1.yml@develop
     with:
       extra_github_env: |
-        GODWOKEN_PREBUILD_IMAGE_NAME=ghcr.io/godwokenrises/godwoken-prebuilds:1.7-rc-202210271055
-        GODWOKEN_KICKER_REF=4892b58fe7157db286000fb831ec8e5cf197c684
-        GODWOKEN_TESTS_REF=55bce8730821c79ffcc0acceb24eb8134feb95cb
+        GODWOKEN_KICKER_REF=develop
+        GODWOKEN_TESTS_REF=develop
```
Comment (keroro520) [inline .github/workflows/gwos-evm-ci.yml]: it is reduandent

**Review by keroro520 — DISMISSED** [review]

**Review by keroro520 — APPROVED** [review]

**Review by zeroqn — APPROVED** [review]

**Review by Flouse — APPROVED** [review]

**Pull request merged**